client/examples/client.rs

```diff
@@ -0,0 +1,33 @@
+//! gRPC client implementation.
+//!
+//! This is an example of a gRPC client that connects to Arrow's pricing
+//! server.
+
+use grpc::pricing_client::PricingClient;
+use grpc::PricingRequest;
+
+///module svc_scheduler generated from svc-scheduler.proto
+pub mod grpc {
```

Comment:
I think you can use the library from the Cargo.toml instead of redeclaring the mod

Reply:
[80260e0](https://github.com/Arrow-air/svc-pricing/pull/1/commits/80260e06accbc6bd971ec428e0038db1df1e4ad5)